components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -47,17 +57,53 @@ export default function EditUserInformationModal({ children }: EditUserInformati
 
   //   TODO: Add error states for each field (post-mvp)
 
-  const successfulEditToast = useToast();
+  const toast = useToast();
 
-  const handleSubmit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    // TODO: Will be addressed in API hookup
-    onClose();
+  // Submit edited user information mutation
+  const [submitEditedUserInformation, { loading }] = useMutation<
+    UpdateApplicantResponse,
+    UpdateApplicantRequest
+  >(UPDATE_APPLICANT_MUTATION, {
+    onCompleted: data => {
+      if (data?.updateApplicant.ok) {
+        onClose();
+        toast({
+          render: () => (
+            <SuccessfulEditAlert>{"User's information has been edited."}</SuccessfulEditAlert>
+          ),
+        });
+      }
+    },
+    onError: error => {
+      onClose();
+      toast({
+        status: 'error',
+        description: error.message,
+      });
+    },
+  });
 
-    successfulEditToast({
-      render: () => (
-        <SuccessfulEditAlert>{"User's information has been edited."}</SuccessfulEditAlert>
-      ),
+  /**
+   * Handle edit submission
+   */
+  const handleSubmit = async (event: SyntheticEvent) => {
+    event.preventDefault();
+    await submitEditedUserInformation({
+      variables: {
+        input: {
+          id: applicantId,
+          firstName: firstName,
+          lastName: lastName,
+          dateOfBirth: dateOfBirth,
+          gender: gender,
+          email: email,
+          phone: phoneNumber,
+          addressLine1: addressLine1,
+          addressLine2: addressLine2,
+          city: city,
+          postalCode: postalCode,
```

Comment:
```suggestion
          firstName,
          lastName,
          dateOfBirth,
          gender,
          email,
          phone:,
          addressLine1,
          addressLine2,
          city,
          postalCode,
```

Reply:
phone should be set to phoneNumber which is set on line 195, but the rest of the fields will be changed